app/views/to_do_items/_new.html.haml

```diff
@@ -0,0 +1,7 @@
+= form_for @item, url: to_do_items_path, method: :post, remote: true, html: {id: "form-#{@item.list_id}"} do |f|
+  .form-group
+    = f.text_field :text, id: "item-input-#{@item.list_id}", class: 'text-input', placeholder: "#{t 'item.add.placeholder'}"
+    = f.hidden_field :list_id, value: @item.list_id
+
+  = f.submit "#{t 'item.add.button'}", id: "add-item-button-#{@item.list_id}", class: 'btn btn-success btn-sm'
+  = f.button "#{t 'item.cancel_button'}", id: "cancel-item-button-#{@item.list_id}", class: 'btn btn-default btn-sm', type: :button
```

Comment:
@vlasiak, I'm wondering whether the following would work:
```ruby
f.submit t('item.add.button')
```
If it does, can you please get rid of `#{}` expression through this PR?